src/prognostic_equations/remaining_tendency.jl

```diff
@@ -46,24 +298,13 @@ NVTX.@annotate function additional_tendency!(Yₜ, Y, p, t)
     thermo_params = CAP.thermodynamics_params(params)
     (; ᶜp, sfc_conditions, ᶜts) = p.precomputed
 
-    vst_uₕ = viscous_sponge_tendency_uₕ(ᶜuₕ, viscous_sponge)
-    vst_u₃ = viscous_sponge_tendency_u₃(ᶠu₃, viscous_sponge)
-    vst_ρe_tot = viscous_sponge_tendency_ρe_tot(ᶜρ, ᶜh_tot, viscous_sponge)
-    rst_uₕ = rayleigh_sponge_tendency_uₕ(ᶜuₕ, rayleigh_sponge)
     hs_args = (ᶜuₕ, ᶜp, params, sfc_conditions.ts, moisture_model, forcing_type)
     hs_tendency_uₕ = held_suarez_forcing_tendency_uₕ(hs_args...)
     hs_tendency_ρe_tot = held_suarez_forcing_tendency_ρe_tot(ᶜρ, hs_args...)
     edmf_cor_tend_uₕ = edmf_coriolis_tendency_uₕ(ᶜuₕ, edmf_coriolis)
     lsa_args = (ᶜρ, thermo_params, ᶜts, t, ls_adv)
     bc_lsa_tend_ρe_tot = large_scale_advection_tendency_ρe_tot(lsa_args...)
 
-    # TODO: fuse, once we fix
-    #       https://github.com/CliMA/ClimaCore.jl/issues/2165
```

Comment:
This is still open in ClimaCore (I don't see any linked PRs, however the thread suggests there may not actually be an issue here; if this has been closed by existing PRs in ClimaCore, or identified as a spurious result, we should close the issue).